eclcli/bare/v2/uefi.py

```diff
@@ -1,6 +1,5 @@
 import json
```

Comment:
I think that it is necessary to describe like other processing.
```suggestion
try:
    import json
except ImportError:
    import simplejson as json
```